service/common/src/main/java/org/apache/polaris/service/ratelimiter/TokenBucketConfiguration.java

```diff
@@ -20,14 +20,9 @@
 
 import java.time.Duration;
 
-public interface RateLimiterConfiguration {
+public interface TokenBucketConfiguration {
 
-  TokenBucketConfiguration tokenBucket();
+  long requestsPerSecond();
 
-  interface TokenBucketConfiguration {
-
-    long requestsPerSecond();
-
-    Duration window();
-  }
+  Duration window();
```

Comment:
nit: Can we use new config doc generating annotations already?

Reply:
Question for @snazy I guess. If so, I can look into it next.